app/containers/TextInput/FormTextInput.tsx

```diff
@@ -116,18 +116,20 @@ export const FormTextInput = ({
 	const Input = bottomSheet ? BottomSheetTextInput : TextInput;
 
 	const inputError = getInputError(error);
-	const accessibilityLabelRequired = required ? `, ${i18n.t('Required')}` : '';
-	const accessibilityInputValue = (!secureTextEntry && value && isIOS) || showPassword ? `, ${value ?? ''}` : '';
-	const accessibilityInputError = i18n.t('Error_prefix', { message: inputError });
+	const accessibilityLabelText = useMemo(() => {
+		const labelText = accessibilityLabel || label || '';
+		const requiredText = required ? `${i18n.t('Required')}.` : '';
+		const valueText = (!secureTextEntry && value && isIOS) || showPassword ? `${value ?? ''}.` : '';
+		const errorText = inputError ? `${i18n.t('Error_prefix', { message: inputError })}.` : '';
+		const a11yLabel = `${labelText ? `${labelText}.` : ''} ${requiredText} ${errorText} ${valueText}`.trim();
```

Comment:
This may get rid of the multiple empty spaces

```suggestion
		const labelText = `${accessibilityLabel || label || ''}`.trim();
		const requiredText = required ? ` ${i18n.t('Required')}.` : '';
		const errorText = inputError ? `${i18n.t('Error_prefix', { message: inputError })}.` : '';
		const valueText = (!secureTextEntry && value && isIOS) || showPassword ? ` ${value ?? ''}.` : '';
		const a11yLabel = `${labelText}${requiredText}${errorText}${valueText}`.trim();
```